wiki/Tournaments/Official_support/en.md

```diff
@@ -117,6 +115,8 @@ Users under an active tournament ban are expected to disclose their tournament b
 
 At the support team's discretion, staff members whose conduct directly results in a tournament losing its official support may not fulfil the same roles in, nor be the host or admin of, another badged tournament for at least 4 months. In the event this cooldown is applied to a user, they will be informed directly through the [tournament team](mailto:tournaments@ppy.sh).
 
+A list of all tournament staff must be publicly visible in an easily accessible location. This list may be contained within the tournament's forum post, website, or spreadsheet.
```

Comment:
and i suppose tc will have to find it instead of being linked to it in the request email?

Reply:
since the forum thread is required in the screening email, this is generally a non-issue in my experience

Reply:
Any list of staff sent in the request email is subject to change. Players, once eliminated, can be enlisted as referees, to give an example. It's more important to have transparency at all times than at a singular point in time (when the email is sent it).

Reply:
On the email they need to link the forum thread where the tournament is announced (or a wiki page / webpage), is part of the requirements for support

Reply:
> It's more important to have transparency at all times than at a singular point in time (when the email is sent it).

that's not what i mean. it may be a slight overhead to check each possible location for the staff list mentioned in this line. it's a matter of convenience.

but if this1neguy says it's alright then ok